README.md

```diff
@@ -1 +1,54 @@
-# Hello-Microverse-Review
\ No newline at end of file
+# Hello-Microverse-Review
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+This is a project for trial review.
+
+## Built With
+
+- HTML
+- CSS
+
+
+
+## Getting Started
+
+**Run you local server to run the app in your browser. For example, you can use visual studio code's live server to run the app**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### clone the app
+
+### Run 'npm install' from the command line
+
+### Run 'npx hint .' to test for html linting check
+
+### Run 'npx stylelint "**/*.{css,scss}"' to test for css linting check 
+
+
+
+## Authors
+
+👤 **Henry Kc**
+
+- GitHub: [@githubhandle](https://github.com/HENRYKC24)
+- Twitter: [@twitterhandle](https://twitter.com/henrykc24)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/henry-kc)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/HENRYKC24/Hello-Microverse/issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to everyone that made this project a success
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
This link is broken Would you kindly fix it? Your [README](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes) is the entry point of your users and having it well written without any broken links gives a good first impression and maximises the potential of your product to your users.

Reply:
I will fix this for sure. Good job, @Kalunge :clap: